examples/website/i3s/app.js

```diff
@@ -214,21 +227,33 @@ export default class App extends PureComponent {
   }
 
   _renderStats() {
+    const {showMemory, sublayers} = this.state;
     // TODO - too verbose, get more default styling from stats widget?
-    return <StatsWidgetContainer ref={(_) => (this._statsWidgetContainer = _)} />;
+    const style = {
+      display: 'flex',
+      top: '125px'
+    }
+    if (!showMemory) {
+      style.display = 'none';
+    }
+    if (sublayers?.length) {
```

Comment:
initialize sublayers with empty array so we don't need the `?.` ?

Reply:
Done